spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    email     "user@example.com"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.